src/workflows/airqo_etl_utils/data_api.py

```
@@ -132,15 +132,13 @@ def get_maintenance_logs(
 
     def get_devices(
         self,
-        network: DeviceNetwork = None,
-        device_category: DeviceCategory = None,
+        params: Dict[str, Any] = {},
```

Ruff (0.14.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)